index.js

```diff
@@ -67,7 +102,20 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+function countByRating(movies) {
+  let ratingsCount = {};
+
+  if(movies.length === 0){
+    return ratingsCount;
+  }
+
+  for (let i = 0; i < movies.length; i ++){
+    let rated = movies[i].rated
+    ratingsCount[rated] = (ratingsCount[rated] + 1 || 1)
+  }
```

Comment:
I love the short circuit.   That being said what does `ratingsCount[rated]+1` resolve to in the event that there is no `ratingsCounts[rated]`